Library/Homebrew/cmd/update.sh

```diff
@@ -378,19 +381,34 @@ EOS
   for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
   do
     [[ -d "$DIR/.git" ]] || continue
+    cd "$DIR" || continue
+
+    TAP_VAR="$(repo_var "$DIR")"
+    declare PREFETCH_REVISION"$TAP_VAR"="$(read_current_revision)"
```

Comment:
@UniqMartin sorted in https://github.com/Homebrew/brew/commit/24b5db53672843647da29073060bff3f81b05b90.